Домашнє завдання №2

1. Скільки різних паролів можна скласти, використовуючи три літери англійського алфавіту, якщо літери не повинні повторюватися?

В даному випадку ми можемо використати формулу P(n,r) = n!/(n-r)!, n = 26 (кількість літер в англійському алфавіті), r = 3

In [13]:
import math

n = 26
r = 3
P = math.factorial(n) // math.factorial(n - r)
print(P)

15600


Відповідь: 15600 різних паролів можна скласти, використовуючи три літери англійського алфавіту, якщо літери не повторюються.

1. Є група з 8 людей. Скількома способами можна обрати комітет із 3 осіб?

В даному випадку ми можемо використати формулу P(n,r) = n!/((n-r)!*r!), оскільки порядок розмщення немає значення, n = 8, r = 3

In [17]:
n = 8
r = 3
P = math.factorial(n) // (math.factorial(n - r) * math.factorial(r))
print(P)

56


Відповідь: 56 способами можна обрати комітет із 3 осіб.

3. Скількома різними способами можна розташувати на полиці 5 різних книг?

В даному випадку ми можемо використати формулу P = n!, n = 5

In [19]:
n = 5
P = math.factorial(n)
print(P)

120


Відповідь: 120 різними способами можна розташувати на полиці 5 різних книг.

4. В урні знаходяться 4 червоні, 3 сині і 2 зелені кулі. Скількома способами можна витягти 3 кулі, якщо вони повинні бути різних кольорів?

варіант № 1: Ми можемо зазстосувати правило добутку: 4*3*2 = 24
В нас буде 24 способа щоб витягти 3 кулі різних кольорів і при цьому порядок їх розміщення має значення.
варіант № 2: Якщо порядок їх розміщення не має значення:
то 24 способи скорочуємо на кількість перестановок 3 куль (3! = 3*2*1 = 6), 24 / 6 = 4

Відповідь 1: 24 способами можна витягти 3 кулі, якщо вони повинні бути різних кольорів і їх розміщення має значення.
Відповідь 2: 4 способами можна витягти 3 кулі, якщо вони повинні бути різних кольорів і їх розміщення не має значення.